SpotifyApp/SpotifyApp/SpotifyTableViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AlbumTableViewController.swift
+//  SpotifyApp
+//
+//  Created by Eashir Arafat on 10/28/16.
+//  Copyright © 2016 Evan. All rights reserved.
+//
+
+import UIKit
+
+class SpotifyTableViewController: UITableViewController {
+   
+    internal let AlbumJSONFileName: String = "SpotifyAlbums.json"
```

Comment:
the names of your variables all start with capital letters! Do not do this ever.